docs.json

```diff
@@ -970,14 +970,6 @@
                         "pages": [
                           "weave/reference/python-sdk/trace_server_bindings/remote_http_trace_server"
                         ]
-                      },
```

Comment:
Removing malformed links from previous regeneration (note the extra `/weave/` path component)